src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModel.java

```diff
@@ -182,7 +182,15 @@ public int getChildCount(Object parent) {
             result = ((DetailNode) parent).getChildren().length;
         }
         else {
-            result = ((DetailAST) parent).getChildCount();
+            if (parseMode == ParseMode.JAVA_WITH_JAVADOC_AND_COMMENTS
+                    && ((AST) parent).getType() == TokenTypes.COMMENT_CONTENT
+                    && JavadocUtils.isJavadocComment(((DetailAST) parent).getParent())) {
+                //Child is javadoc tree
```

Comment:
It is similar to hardcode in ASTTreePrinter. AST and javadoc tree are different trees with different classes of nodes. ((DetailAST) parent).getChildCount() returns 0 because COMMENT_CONTENT token hasn't children.